### ISSUE TITLE
ErrorException when symbols are imported

### ISSUE THREAD

**Issue by diegozea** [tonyhffong/Lint.jl#135]
Example:

``` julia
julia> import Base: ==, hash, length, size, -, +, ./, norm, dot, angle, cross, vec, any, print, show, parse

julia> lintstr("import Base: ==, hash, length, size, -, +, ./, norm, dot, angle, cross, vec, any, print, show, parse")
ErrorException("syntax: incomplete: premature end of input")
import Base.==
ErrorException("syntax: incomplete: premature end of input")
import Base.-
ErrorException("syntax: incomplete: premature end of input")
import Base.+
ParseError("invalid operator \"../\"")
import Base../
0-element Array{Lint.LintMessage,1} 

```

**Comment by TotalVerb** [comment]
I think this has to do with the strange eval-ing of imported modules. Will take a look later today.

**Comment by TotalVerb** [comment]
I think this might have been fixed by #132. Can you do

```julia
julia> Pkg.checkout("Lint")
```

and see if the problem persists?

**Comment by diegozea** [comment]
Thank you! It's solved.
